tests/sources/python-config-test.py

```diff
@@ -78,6 +78,7 @@
                 exit(1)
 
 ### Validate libreadline
-if not have_libreadline:
-    print('Missing libreadline')
-    exit(1)
+if sys.version_info < (3, 12):
```

Comment:
What is the reason for disabling this test for python 3.12 and higher? I compared docs for python [3.11.0](https://docs.python.org/3.11/using/configure.html) and [3.12.0a1](https://docs.python.org/3.12/using/configure.html) and found out that both these versions should contain [readline module](https://docs.python.org/3.12/library/readline.html#module-readline) by default. Isn't it just the bug of 3.12.0a1 and it doesn't have this module?

Reply:
The answer was obtained in an internal conversation. I'm resolving this conversation.